Sponge/src/Variables.java

```diff
@@ -28,6 +29,10 @@ public Text replaceVariables(Player p, String str) {
 			address = p.getConnection().getAddress().getAddress().toString();
 			address = address.replaceAll("/", "");
 		}
+		DecimalFormat tpsformat = new DecimalFormat("#0.00");
+		if (str.contains("%tps%")) {
+			str = str.replace("%tps%", String.valueOf(tpsformat.format(Sponge.getServer().getTicksPerSecond())));
```

Comment:
`DecimalFormat#format` returns string, so you don't need another `String.valueOf`

Reply:
Yeah your right my apologies.